src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -4101,27 +4100,25 @@ namespace winrt::TerminalApp::implementation
 
             TitlebarBrush(acrylicBrush);
         }
-        else if (theme.TabRow())
+        else if (const auto tabRowBg{ _activated ? theme.TabRow().Background() :
```

Comment:
This call is causing a crash on startup in the current build of main with the error `Access violation reading location 0x0000000000000000` (just did a git pull, clean and tried to run)

Reply:
Ah, I think the `theme.TabRow() != nullptr` check should come before we try to access the `Background`?